app/uk/gov/hmrc/soletraderidentificationfrontend/services/AuditService.scala

```diff
@@ -81,6 +81,7 @@ class AuditService @Inject()(auditConnector: AuditConnector, soleTraderIdentific
       optIdentifiersMatch <- soleTraderIdentificationService.retrieveIdentifiersMatch(journeyId)
       optAuthenticatorResponse <-
         optIdentifiersMatch match {
+          case Some(_) if optNino.isEmpty => Future.successful(None)
           case Some(true) =>
```

Comment:
I know it's not on this PR, but could we name this boolean rather than matching on true, it just makes it clearer.

e.g.
case Some(identifiersMatch) if identifiersMatch => ???

Is the failure case on this meant to cover Some(false) only if nino is populated?